.github/workflows/main.yml

```diff
@@ -24,19 +35,24 @@ jobs:
           compiler_version: "10"
           python: 3.9
           cmake_config: -DMATERIALX_BUILD_SHARED_LIBS=ON -DMATERIALX_BUILD_MONOLITHIC=ON
+          build_mdl_sdk: OFF
```

Comment:
Can we omit the explicit setting of `build_mdl_sdk` to `OFF`, as is done for settings such as `test_render` and `coverage_analysis`?  Ideally, we'd only need to state when this is set to `ON`, in order to keep our job matrix as clear and readable as possible.